src/commands/solidarity.ts

```diff
@@ -48,11 +62,18 @@ namespace Solidarity {
     await Promise.all(checks)
       .then(results => {
         const errors = reject(isNil, flatten(results))
+        const silentOutput = context.outputMode == SolidarityOutputMode.SILENT
```

Comment:
one nit.  I always use triple equals.   Just because i protest the idea of `==`.  I suggest you protest with me :)  Gonna merge this and fix in a subsequent PR.